test/cmdlineTests/too_long_line_edge_out.sol.err

```diff
@@ -1,6 +1,13 @@
-too_long_line_edge_out.sol:1:1: Warning: Source file does not specify required compiler version!
-contract C {
-^ (Relevant source part starts here and spans across multiple lines).
-too_long_line_edge_out.sol:2:37: Error: Identifier not found or not unique.
- ...   function fffffffffffffffffffffff(announcementTypeTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT Typ, string A) onlyOwner external  ...
-                                        ^----------------------------------------------------------------------------------------------^
+
+Warning: Source file does not specify required compiler version!
+
+ --> too_long_line_edge_out.sol:1:1: Source file does not specify required compiler version!
```

Comment:
Do we really need to repeat the message?

Reply:
no. this output just hasn't been updated to the code changes yet. will fix that. thanks :)